scripts/homework4.py

```diff
@@ -0,0 +1,125 @@
+# Author: Shad Fernandez
+# 2021-OCT-11
+# BDA 696 HW4
+
+import sys
+
+import binresponsebypredictor as bin
+import pandas as pd
+import plot_pred_response as prr
+import statsmodels.api as stats
+
+
+def main():
+    df = pd.read_csv("../datasets/Iris.csv")
+    print(df.columns)
+    response_col = str(input("Please enter the column name of the response variable: "))
```

Comment:
It may be cleaner to just have the user pass in the response column as a string variable in your main function rather than take input from the user